src/components/recruit/RecruitItem.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from "styled-components";
+
+type RecruitItemComponentProps = {
+  id: number;
+  name: string;
+  description: string;
+  from: Date | null;
+  to: Date | null;
+};
+
+export function RecruitItem({
+  id,
+  name,
+  description,
+  from,
+  to,
+}: RecruitItemComponentProps) {
+  // to가 undefined이면 상시 모집이므로 항상 활성화
+  const isActive = to ? to.getMilliseconds() > Date.now() : true;
+
+  return (
+    <Container href={`/recruiting/${id}`} $isActive={isActive}>
+      <RecruitNameArea>
+        <RecruitName>{name}</RecruitName>
+        <RightArrow src="/image/rightAngleBracket.svg" />
+      </RecruitNameArea>
+      <RecruitDescription>
+        {/* to가 없을 때만 상시모집, from이 없을 경우에는 빈칸으로 둔다. */}
+        {to
+          ? `${from?.toISOString().split("T")[0] ?? ""} ~ ${
+              to.toISOString().split("T")[0]
+            }`
```

Comment:
소소한 팁: 이렇게 쓰면 좀더 보기 편할 수 있습니다 (아님말고)

```suggestion
          ? [
              from?.toISOString().split("T")[0] ?? "",
              to.toISOString().split("T")[0],
            ].join(' ~ ')
```